requirements.txt

```
@@ -1,5 +1,5 @@
 # torch>=2.1.0dev
-lightning @ git+https://github.com/Lightning-AI/lightning@master
+lightning @ git+https://github.com/Lightning-AI/lightning@carmocca/xlafsdp-ckpt-fixes
```

Reminder to undo before merge

```suggestion
lightning @ git+https://github.com/Lightning-AI/lightning@master
```